package.json

```diff
@@ -2,23 +2,25 @@
   "name": "string-ts",
   "version": "1.2.0",
   "description": "Strongly-typed string functions.",
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
+  "main": "dist/commonjs/index.js",
+  "module": "dist/es/index.js",
+  "jsnext:main": "dist/es/index.js",
```

Comment:
```suggestion
  "jsnext:main": "dist/es/index.js",
   "sideEffects": false,
```

Reply:
I've read that adding this property also helps on tree-shaking as it indicates nothing in this package modifies global variables